### DISCUSSION TITLE
Is Tiled VAE still working on AUTOMATIC1111 webui?

### DISCUSSION THREAD

**Discussion by RIPBon** [pkuliyi2015/multidiffusion-upscaler-for-automatic1111#301] [Q&A]
I've used this combination before, this summer, it helps me to generate large images like 1536x1024 since I have AMD Radeon RX 5500 XT card and so I can't generate such large images on vanilla Stable Diffusion. But two weeks ago I had to reinstall SD and Tiled VAE and now it feels like it doesn't work at all with the current version of AUTOMATIC1111 because I keep getting "CUDA out of memory" errors and I see no difference in SD actions and errors with and without Tiled VAE on.

Am I doing something wrong or is Tiled VAE having problems with the current version of AUTOMATIC SD?

**Comment by cgessai** [comment]
Curious to know the answer to this as well.  Installed it for the first time and having problems with it.

**Comment by LindezaBlue** [comment]
as of right now I am having issues with it on the dev branch and forge which both are running ver 1.8.0, while the release version is 1.7.0.
